README.md

```diff
@@ -11,3 +11,6 @@ To get started:
   - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
 
 Then follow the on-screen instructions to set up your repo.
+
```

Comment:
Can you have this change (and the pom.xml change) in a different PR? Try not to have unrelated changes in the same PR.

Reply:
I have made a new PR